client/landing/gutenboarding/utils/domain-suggestions.ts

```diff
@@ -1,37 +1,5 @@
-/**
- * External dependencies
- */
-import type { DomainSuggestions } from '@automattic/data-stores';
-
-type DomainSuggestion = DomainSuggestions.DomainSuggestion;
+type DomainSuggestion = import('@automattic/data-stores').DomainSuggestions.DomainSuggestion;
```

Comment:
We moved away from importing types like this in [#42804](https://github.com/Automattic/wp-calypso/pull/42804).
See this exact type import change:
https://github.com/Automattic/wp-calypso/pull/42804/files#diff-9fb3f50c91d02f132bfab858e87cb792R10-R35